App/Components/PhotoGrid.js

```diff
@@ -4,7 +4,8 @@ import {
   Image,
   Text,
   FlatList,
-  TouchableOpacity
```

Comment:
Any reason we want to update `PhotoGrid.js` instead of put the component from SB in the SB folder like we were doing for many of the others? Just trying to make it easier to clean up later when we want to remove all unused/old code.

Reply:
Just didn't see a ton of gain going that direction on this one